docs/sofapython3/tutorials/Tripod/details/tripod.py

```diff
@@ -103,66 +54,115 @@ def __attachToActuatedArms(self, radius=60, numMotors=3, angleShift=180.0):
+    scene = Scene(rootNode, gravity=[0.0, -9810, 0.0], iterative=False,
+                  plugins=['SofaSparseSolver', 'SofaOpenglVisual', 'SofaSimpleFem', 'SofaDeformable', 'SofaEngine',
+                           'SofaGeneralRigid', 'SofaMiscMapping', 'SofaRigid', 'SofaGraphComponent', 'SofaBoundaryCondition',
+                           'SofaGeneralAnimationLoop', 'SofaConstraint'])
     scene.addMainHeader()
-    scene.addObject('AttachBodyButtonSetting', stiffness=10)  # Set mouse spring stiffness
     scene.addObject('DefaultVisualManagerLoop')
     scene.addObject('FreeMotionAnimationLoop')
     scene.addObject('GenericConstraintSolver', maxIterations=50, tolerance=1e-5)
     scene.Simulation.addObject('GenericConstraintCorrection')
-
+    scene.Settings.mouseButton.stiffness = 10
+    scene.Simulation.TimeIntegrationSchema.rayleighStiffness = 0.05
     scene.VisualStyle.displayFlags = "showBehavior"
+    scene.dt = 0.01
+
+    # Add the blueprint
+    scene.Modelling.addChild(Blueprint())
 
+    # Add the tripod
     tripod = scene.Modelling.addChild(Tripod())
+    tripod.BoxROI0.drawBoxes = True
+    tripod.BoxROI1.drawBoxes = True
+    tripod.BoxROI2.drawBoxes = True
 
-    scene.Simulation.addChild(tripod.RigidifiedStructure)
+    # Use this to activate some rendering on the rigidified object ######################################
+    setData(tripod.RigidifiedStructure.RigidParts.dofs, showObject=True, showObjectScale=10, drawMode=2)
+    setData(tripod.RigidifiedStructure.RigidParts.RigidifiedParticules.dofs, showObject=True, showObjectScale=1,
+            drawMode=1, showColor=[1., 1., 0., 1.])
+    setData(tripod.RigidifiedStructure.DeformableParts.dofs, showObject=True, showObjectScale=1, drawMode=2)
+    #####################################################################################################
 
-    motors = scene.Simulation.addChild("Motors")
-    for i in range(3):
-        motors.addChild(tripod.getChild('ActuatedArm'+str(i)))
+
+    scene.Simulation.addChild(tripod)
+
+    def myanimate(targets, factor):
+        for arm in targets:
+            arm.angleIn = -factor * math.pi / 4.
+
+    animate(myanimate, {"targets": [tripod.ActuatedArm0, tripod.ActuatedArm1, tripod.ActuatedArm2]}, duration=1)
 
     # Temporary additions to have the system correctly built in SOFA
     # Will no longer be required in SOFA v22.06
-    scene.Simulation.addObject('MechanicalMatrixMapper',
-                                 name="mmmFreeCenter",
-                                 template='Vec3,Rigid3',
-                                 object1="@RigidifiedStructure/DeformableParts/dofs",
-                                 object2="@RigidifiedStructure/FreeCenter/dofs",
-                                 nodeToParse="@RigidifiedStructure/DeformableParts/ElasticMaterialObject")
-
```

Comment:
It's not that easy to follow the changes on github, so just in case, could this be a mistake? I don't see this coupling elsewhere.

Reply:
I was not sure the freeCenter was needed in the createScene part of the Tripod as it is not using inverseModel. Is this the case ?

Reply:
Well if the rigid (freeCenter) is there I think the coupling is needed

Reply:
Yep I've just tried the scene and it's needed